index.d.ts

```diff
@@ -266,3 +266,5 @@ export function isLoop(test: any): boolean;
 export function getModel<S>(loop: S | Loop<S>): S;
 
 export function getCmd(a: any): CmdType | null;
+
+export type HandlerReturn<S> = S | Loop<S>;
```

Comment:
should we call it ReducerHandlerReturn? Might make it more clear

Reply:
Its kinda wordy but I'm impartial

Reply:
The most idiomatic is probably `LoopReducerHandlerFnReturn`, though I agree with Baruch that we're getting 'wordy. `HandlerFnReturn`? `HandlerFunctionReturn`? I dunno. I'm mostly impartial. Maybe a comment above the type would help?

Reply:
Well i thought the name change was more helpful for the user of the library, not the author. Maybe let's leave it and we can change it if there's feedback